User/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+class User(models.Model):
+    first_name         = models.CharField(max_length = 32)
+    last_name          = models.CharField(max_length = 32)
+    email              = models.EmailField()
+    password           = models.BinaryField(null=True)
+    is_registered_user = models.BooleanField()
+    phone_number       = models.CharField(max_length = 32)
+
+    class Meta:
+        db_table = 'users'
+
+class CreditCard(models.Model):
+    user              = models.ForeignKey(User, on_delete=models.CASCADE)
+    creditcard_name   = models.CharField(max_length = 32)
+    creditcard_number = models.CharField(max_length = 32)
+    creditcard_month  = models.IntegerField()
+    creditcard_year   = models.IntegerField()
```

Comment:
```suggestion
    expiration_date  = models.DateField()
```
고객에게 카드의 년/월을 따로 입력받지만, 데이터를 관리하는 입장에서는 사실 날짜 하나에 해당한다고 생각하시는 것이 좋습니다. 또한, 날짜로 기입을 해두어야 유효기간이 지난 카드인지 아직 사용해도 괜찮은 카드인지 판단할 수 있기 때문에 오늘 날짜를 기준으로 연산을 할 수 있는 날짜 필드를 선택하시는 것이 좋습니다. 아래 스택오버플로우가 도움이 될 것 같아요 :) 👍  

https://stackoverflow.com/questions/5695810/how-do-i-store-a-credit-cards-expiration-date-if-it-only-consists-of-the-month